bundle/compliance/cis_gcp/rules/cis_6_6/rule.rego

```diff
@@ -0,0 +1,20 @@
+package compliance.cis_gcp.rules.cis_6_6
+
+import data.compliance.lib.common
+import data.compliance.policy.gcp.data_adapter
+import future.keywords.if
+
+finding = result if {
+	data_adapter.is_sql_instance
+
```

Comment:
You should add a filter.

>2. For every instance of type instanceType: CLOUD_SQL_INSTANCE with backendType: SECOND_GEN, get detailed configuration. Ignore instances of type READ_REPLICA_INSTANCE because these instances inherit their settings from the primary instance. Also, note that first generation instances cannot be configured to have a private IP address.

Reply:
added `is_clous_sql_instance_second_gen`

Reply:
cool, what about:
> Ignore instances of type READ_REPLICA_INSTANCE

Reply:
by only accepting `instanceType: CLOUD_SQL_INSTANCE` we're effectively ignoring `instanceType: READ_REPLICA_INSTANCE`, no ?

Reply:
added a replica to check if checking for `instanceType` is sufficent and it is: 
<img width="343" alt="Screenshot 2023-08-09 at 18 23 35" src="https://github.com/elastic/csp-security-policies/assets/20814186/681c02b6-0d60-428e-af73-1c0c575541a2">